codespell_lib/data/dictionary.txt

```diff
@@ -20181,7 +20185,10 @@ increament->increment
 increas->increase
 incredable->incredible
 incremantal->incremental
+incremeant->increment
```

Comment:
Add `incremeants`?

Reply:
Thanks a lot for the suggestion. Added in 28d466750b178824e86dd2bdbf0ba82f69ab465b